src/interceptors/request.ts

```diff
@@ -74,6 +74,7 @@ export class CacheRequestInterceptor<D>
 
       const oldValidate = config.validateStatus;
       config.validateStatus = (status: number): boolean => {
+        /* istanbul ignore next 'for some reason this code is not detected' */
```

Comment:
this is not used at all in tests due to mock using adapter config. validateStatus is used in the default axios adapter :) 

to add tests on that would require using a real http server and querying to it. (start a node http server on a random port, and then use it in tests), however that's a little bit overkill for this MR, you might want to do it in the future though

Reply:
Exactly, but I'm thinking about adding this in the future as it looks like it can give the project more confidence.